src/main/java/seedu/address/logic/commands/AddEventCommand.java

```diff
@@ -1,20 +1,16 @@
 package seedu.address.logic.commands;
 
-import static java.util.Objects.requireNonNull;
```

Comment:
Need to fix. Do refer to [this](https://se-education.org/addressbook-level3/SettingUp.html#configurations-to-do-before-writing-code).

Reply:
Resolved in abf2cb4.